mir_eval/separation.py

```diff
@@ -114,6 +108,13 @@ def validate(reference_sources, estimated_sources):
                                                 MAX_SOURCES))
 
 
+def _any_source_silent(sources):
+    """Returns true if the parameter sources has any silent first dimensions"""
+    return (np.any(np.all(np.sum(sources,
+                                 axis=tuple(range(2, sources.ndim))) == 0,
+                          axis=1)))
```

Comment:
Since you asked about code style (otherwise I would not nitpick this... probably), I think it's much cleaner to do
```Python
return np.any(np.all(np.sum(
    sources, axis=tuple(range(2, sources.ndim))) == 0, axis=1))
```
Beyond that, I think it should be about-as-fast and slightly cleaner to use `np.arange` rather than `tuple(range)` in this case:
```Python
return np.any(np.all(np.sum(
    sources, axis=np.arange(2, sources.ndim)) == 0, axis=1))
```